src/material-experimental/mdc-chips/chip-option.html

```diff
@@ -1,7 +1,7 @@
-<ng-content select="[mat-chip-avatar], [matChipAvatar]"></ng-content>
+<ng-content select="mat-chip-avatar, [matChipAvatar]"></ng-content>
 <svg *ngIf="_chipListMultiple" class="mdc-chip__checkmark-svg" viewBox="-2 -3 30 30">
   <path class="mdc-chip__checkmark-path" fill="none" stroke="black"
     d="M1.73,12.91 8.1,19.28 22.79,4.59"/>
 </svg>
 <div class="mdc-chip__text"><ng-content></ng-content></div>
-<ng-content select="[mat-chip-trailing-icon],[matChipRemove],[matChipTrailingIcon]"></ng-content>
\ No newline at end of file
+<ng-content select="[mat-chip-trailing-icon],[matChipRemove],[matChipTrailingIcon]"></ng-content>
```

Comment:
Do you need to remove the square brackets around [mat-chip-trailing-icon] as well?

Reply:
Yes good catch, just made that change as well